lib/DeonRestClient.ts

```diff
@@ -165,13 +165,21 @@ class DeonRestClient implements DeonApi {
   };
 
   csl: CslApi = {
-    check: (i: CheckExpressionInput) => this.http.post('/csl/check', i)
-      .then(r => r.ok ? [] : r.json()),
-
-    checkExpression: (i: CheckExpressionInput, id?: string) => {
-      return this.http.post(`/csl/check-expression${id != null ? `/${id}` : ''}`, i)
-        .then(r => r.ok ? [] : r.json());
-    },
+    check: (i: CheckExpressionInput) => this.http
+      .post('/csl/check', i)
+      .then((r) => {
```

Comment:
Could we name this handler? Seems we're doing the same thing twice

Reply:
Alright.